frontend/views/components/BannerScoped.vue

```diff
@@ -0,0 +1,106 @@
+<template lang='pug'>
+  transition-expand
+    .c-container(v-if='ephemeral.text')
+      banner-simple(class='c-banner' :severity='ephemeral.severity')
+        .c-inner
+          .c-inner-text(:data-test='dataTest') {{ ephemeral.text }}
+          button.is-icon-small.c-button(
+            type='button'
+            :class='`is-${ephemeral.severity}`'
+            :aria-label='L("Dismiss message")'
+            @click='clean'
+          )
+            i.icon-times
+</template>
+
+<script>
+import BannerSimple from '@components/BannerSimple.vue'
+import TransitionExpand from '@components/TransitionExpand.vue'
+
+export default {
+  name: 'BannerScoped',
+  components: {
+    BannerSimple,
+    TransitionExpand
+  },
+  props: {
+    dataTest: {
+      type: String,
+      default: 'feedbackMsg'
+    }
```

Comment:
Perhaps good to pass in all the attributes [using `$attrs`](https://vuejs.org/v2/api/#vm-attrs) instead, so that `data-test` can be used directly on this component just the same way it is used everywhere else, and new developers don't have to wonder, "Why `dataTest` here and `data-test` there?"

Reply:
I don't agree with $attrs in this case because the data-test is not appended to the parent level but instead on of the childrens. Nevertheless, people can write `data-test` when using the component because [props interpret any uppercase characters as lowercase ](https://vuejs.org/v2/guide/components-props.html#Prop-Casing-camelCase-vs-kebab-case).

Reply:
Well in that case can you update the code to write `data-test` consistently everywhere?

Reply:
I already did that :)